composer.json

```diff
@@ -8,8 +8,10 @@
     "symplify/easy-coding-standard": "9.3.12"
   },
   "require-dev": {
+    "composer/composer": "2.*",
```

Comment:
Btw, funnt thing. I thought that I can remove that dependency, but without it I cannot run Composer binary. :man_shrugging: